GSA/digital-gov-drupal
# Feature/digital 95 install breadcrumbs

## Jira ticket

[DIGITAL-95](https://cm-jira.usa.gov/browse/DIGITAL-95)

## Purpose

Breadcrumbs in Drupal should act the same way they do on the existing site. This PR covers these content types:

Resources
Blogs
Events
Communities

To test Topics, go to resources.
To test Resources, go to Topics and Resources.

Other breadcrumb items will be set up later.

## Deployment and testing

To test:
1. pull branch
2. import configuration (drush cim)
3. import content (lando si)
4. Make sure the breadcrumbs appear as they do (for these content types) at http://www.digital.gov:
        - All breadcrumbs are the same, including if they are not there at all.
        - All URLs are the same.
       
To make testing easier, it might be a good idea to have digital.gov up while you're testing against local. I put this into a spreadsheet, if it helps, here it is:

[digital.gov_breadcrumb_links.xlsx](https://github.com/user-attachments/files/17835036/digital.gov_breadcrumb_links.xlsx)

### Local Setup



### QA/Testing instructions



## Checklist for the Developer




## Checklist for the Peer Reviewers
